core/drivers/gic.c

```diff
@@ -454,12 +493,22 @@ void gic_it_handle(struct gic_data *gd)
 	gic_write_eoir(gd, iar);
 }
 
+#ifndef CFG_CORE_WORKAROUND_ARM_NMFI
+/* Override itr_core_handler() with core interrupt controller implementation */
+void itr_core_handler(void)
+{
+	gic_native_itr_handler();
```

Comment:
Why not open code `gic_native_itr_handler()` here?

Reply:
what do you mean by "open"?

Reply:
Copy the content of `gic_native_itr_handler()`.

Reply:
Because of `#ifndef CFG_CORE_WORKAROUND_ARM_NMFI`, I had to tag `gic_read_iar()` and `gic_write_eoir()` as `__maybe_unused`. I prefer the way implemented here where only `gic_native_itr_handler()` is tagged `__maybe_unused`. This way, GIC drvier handler function has a nice `gic_` prefixed label.
That said, I have no strong opinon here.

Reply:
OK, let's keep it as it is then.